api/tests/setup/mixins/create_user_mixin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.auth.models import AbstractUser
+from django.test import TestCase
+
+from ..accounts.creation_helpers import create_user
+
+
+class CreateUserMixin(TestCase):
+    user: AbstractUser
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.user = create_user()
```

Comment:
The concept of a "Mixin" is quite important in Django apps. Unlike other OOP languages like Java, Python allows you to inherit from multiple different classes. This mixin creates a user for each test that is run, meaning that for tests where we need a pre-created user in scope, we can inherit from `CreateUserMixin` and `IntegrationTestCase` (or `UnitTestCase`) to get the setup from both. It's also a good idea to add type hints for the instance variables you create, such as `user: AbstractUser` so that your IDE can offer better completion